src/Microsoft.CodeQuality.Analyzers.Exp/Core/FlowAnalysis/Analysis/CopyAnalysis/CopyAnalysis.CopyDataFlowOperationVisitor.cs

```diff
@@ -249,16 +244,12 @@ protected override CopyAbstractValue ComputeAnalysisValueForOutArgument(Analysis
             
             #endregion
 
-            // TODO: Remove these temporary methods once we move to compiler's CFG
-            // https://github.com/dotnet/roslyn-analyzers/issues/1567
-            #region Temporary methods to workaround lack of *real* CFG
             protected override CopyAnalysisData MergeAnalysisData(CopyAnalysisData value1, CopyAnalysisData value2)
```

Comment:
We need to retain the Merge and Clone helpers for predicate analysis, all the remaining temporary methods have been removed.